include/coap2/net.h

```diff
@@ -347,12 +347,16 @@ coap_context_set_pki_root_cas(coap_context_t *context,
  * Set the context keepalive timer for sessions.
  * A keepalive message will be sent after if a session has been inactive,
  * i.e. no packet sent or received, for the given number of seconds.
- * For reliable protocols, a PING message will be sent. If a PONG has not
- * been received before the next PING is due to be sent, the session will
- * considered as disconnected.
+ * For unreliable protocols, an CoAP Empty message will be sent. If a 
+ * CoAP RST is not received, the CoAP Empty messages will get resent based
+ * on the Confirmable retry parameters until there is a failure timeout,
+ * at which point the session will considered as disconnected.
```

Comment:
"will considered" → "will be considered"

Reply:
Fixed